processor/src/trace/mod.rs

```diff
@@ -169,7 +169,7 @@ impl ExecutionTrace {
         }
     }
 
-    #[cfg(test)]
+    #[cfg(feature = "internals")]
```

Comment:
Shouldn't we follow the pattern of the others and activate this part if we have either test or the feature?

Reply:
We can set this to `#[cfg(any(test, feature = "internals"))]` - but only after `pub` version or process enabled for tests as well.